__tests__/api.test.js

```diff
@@ -202,3 +202,64 @@ describe("POST/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("PATCH/api/articles/:article_id", () => {
```

Comment:
Good testing in the main. 👍 One further test to consider would be if the client uses an invalid id in the url eg `/api/articles/three`